src/ui/screens/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,192 @@
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutAll
+
+                    anchors.fill: parent
+                    anchors.topMargin: VPNTheme.theme.vSpacing
+                    anchors.bottomMargin: VPNTheme.theme.vSpacing
+                    anchors.leftMargin: VPNTheme.theme.windowMargin
+                    anchors.rightMargin: VPNTheme.theme.windowMargin
+                    spacing: VPNTheme.theme.vSpacingSmall
+
+                    VPNTutorialList {
+                        Layout.fillWidth: true
+
+                        customTutorialFilter: (addon) => addon.highlighted
+                    }
+
+                    VPNTutorialCard {
+                        objectName: "featureTourCard"
+
+                        Layout.preferredHeight: VPNTheme.theme.tutorialCardHeight
+                        Layout.fillWidth: true
+
+                        imageSrc: "qrc:/ui/resources/sparkling-check.svg"
+                        imageBgColor: "#2B2A33"
+                        title: VPNl18n.TipsAndTricksFeatureTourCardTitle
+                        description: VPNl18n.TipsAndTricksFeatureTourCardDescription
+                        onClicked: featureTourPopup.startTour();
+                    }
+
+                    Repeater {
+                        model: tipsSections.count
+
+                        Column {
+                            property var section: tipsSections.get(index)
+
+                            visible: guidesList.count || tutorialsList.count
+
+                            Layout.fillWidth: true
+                            Layout.alignment: Qt.AlignTop
+                            Layout.topMargin: VPNTheme.theme.vSpacingSmall
+                            spacing: VPNTheme.theme.vSpacing
+
+                            VPNGuideList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: guidesList
+                                title: section.title
+                                description: section.description
+                                customGuideFilter: section.filter
+                            }
+
+                            VPNTutorialList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: tutorialsList
+                                customTutorialFilter: section.filter
+                            }
+                        }
+                    }
+                }
+            },
+
+            // Tutorials
+            VPNFlickable {
+                flickContentHeight: layoutTutorial.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutTutorial
```

Comment:
Don't really need a ColumnLayout for one component.

You can remove this and just have something like:

```
                    VPNTutorialList { 
                        id: layoutTutorial

                        anchors.top: parent.top
                        anchors.left: parent.left
                        anchors.right: parent.right
                        anchors.topMargin: VPNTheme.theme.vSpacing
                        anchors.leftMargin: VPNTheme.theme.windowMargin
                        anchors.rightMargin: VPNTheme.theme.windowMargin
                    }
```